modules/users/server/controllers/users/users.password.server.controller.js

```diff
@@ -138,6 +138,10 @@ exports.reset = function(req, res, next) {
 									message: errorHandler.getErrorMessage(err)
 								});
 							} else {
+							    // Remove sensitive data before login
+							    user.password = undefined;
```

Comment:
Right, although @almegdad did point out that later on they are sent back to the user in res.json() (line 149).